### PR TITLE
Use fake trigger time or source time for fake report rate limit

### PR DESCRIPTION
**Pull request by linnan-github**
***
<a href="https://pr-preview.s3.amazonaws.com/linnan-github/conversion-measurement-api/pull/676.html" title="Last updated on Jan 19, 2023, 3:33 PM UTC (0f37631)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/WICG/attribution-reporting-api/676/bdfa8a8...linnan-github:0f37631.html" title="Last updated on Jan 19, 2023, 3:33 PM UTC (0f37631)">Diff</a>

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -1065,11 +1065,15 @@ To <dfn>process an attribution source</dfn> given an [=attribution source=] |sou
 1. [=list/Remove=] all [=attribution rate-limit records=] |entry| from the [=attribution rate-limit cache=] if the result of running
     [=can attribution rate-limit record be removed=] with |entry| and |source|'s [=attribution source/source time=] is true.
 1. If |source|'s [=attribution source/randomized response=] is not null and is a [=set=]:
+    1. Let |triggerTime| be null.
     1. [=set/iterate|For each=] [=trigger state=] |triggerState| of |source|'s
         [=attribution source/randomized response=]:
         1. Let |fakeReport| be the result of running [=obtain a fake report=]
             with |source| and |triggerState|.
         1. [=set/Append=] |fakeReport| to the [=event-level report cache=].
+        1. If |triggerTime| is null or |fakeReport|'s [=event-level report/trigger time=] is less than |triggerTime|,
```
Comment (apasel422) [inline index.bs:1074]: Just wondering what the reasoning is for using the minimum trigger time.

Comment (linnan-github) [reply index.bs:1074]: I don't have strong reasoning actually, any argument for using other value?

Comment (apasel422) [reply index.bs:1074]: I'm trying to make sure there's no negative privacy implication, however we choose.

Comment (linnan-github) [reply index.bs:1074]: As confirmed by @badih, we should use source time.

**Pull request closed without merging**